crates/re_data_source/src/data_loader/loader_external.rs

```diff
@@ -142,29 +163,44 @@ impl crate::DataLoader for ExternalLoader {
                     }
                 };
 
-                if !status.success() {
+                let is_compatible =
+                    status.code() != Some(crate::EXTERNAL_DATA_LOADER_NOT_SUPPORTED_EXIT_CODE);
```

Comment:
This will interpret `None` as "Compatible loaded" - is that correct?

Reply:
For now I've made the assumption that plugins are compatible until proven otherwise -- so yes.

I'll add a note.